### PR TITLE
ACS-92: Update openjdk to 11.0.7 and fix vulnerabilites

### PR DESCRIPTION
**Pull request by svidrascu**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -10,6 +10,7 @@ LABEL org.label-schema.schema-version="1.0" \
     org.label-schema.vendor="Alfresco"
 
 RUN yum -y update \
+    python-2.7.5-86.el7 \
```
Comment (jamalkm) [inline Dockerfile:13]: Can't you update to the latest?  E.g. 0:2.7.5-88.el7

File: Dockerfile [hunk 2]
```diff
@@ -30,6 +30,7 @@ RUN yum -y update \
     binutils-2.27-41.base.el7_7.1 \
     curl-7.29.0-54.el7 \
     sqlite-3.7.17-8.el7_7.1 \
+    bash-4.2.46-33.el7 \
```
Comment (jamalkm) [inline Dockerfile:33]: Can't you update to the latest?

File: Dockerfile (hunk 1 above)
Comment (svidrascu) [reply Dockerfile:13]: 86 is latest available sadly, as for all the others .

thats all we get from official centos.

File: Dockerfile (hunk 2 above)
Comment (svidrascu) [reply Dockerfile:33]: same here.

**Pull request merged**